pagefaultgames/pokerogue
# [BUG] Fixes Sketch copying the first move used by the opponent instead of the last

## What are the changes?

Sketch currently copies the first move used in a turn instead of the most recent.
#2758 exposed this issue, but it is not due to switching out like the issue states. Instead, it just always copies the first move used in the opponent's history.

## Why am I doing these changes?



Fixes a bug with Sketch.

## What did change?


SketchAttr now gets the last move used by the opponent, instead of the entire move history.

### Screenshots/Videos


Before fix (copies first move used - Gust)

https://github.com/pagefaultgames/pokerogue/assets/62030095/7059caba-cf6c-40ac-bccc-a7b0378f811f

After fix (copies last move used - Absorb)

https://github.com/pagefaultgames/pokerogue/assets/62030095/71cd14ec-b748-4532-9f4e-010ab1d97ff6

## How to test the changes?




Give opponent different moves, select Smeargle as your starter (preferably with other moves if Smeargle is faster than opponent). Let opponent use different moves. Smeargle uses Sketch on opponent. Smeargle should copy latest move used.

## Checklist
